contrib/runners/python_runner/python_runner/python_action_wrapper.py

```diff
@@ -359,6 +359,12 @@ def _get_action_instance(self):
 
         stdin_data = sys.stdin.readline().strip()
 
+        if not stdin_data:
+            # This could indicate that parent process (e.g. process which runs the tests has
+            # incorrectly opened the stdin and that one is then inherited by the process which is
+            # spawning it which will cause issues)
+            raise ValueError("Received no valid parameters data from sys.stdin")
```

Comment:
Not a blocker, but noting this in the future: it would be a good idea to log what already has been read into `stdin_data`.

```suggestion
            raise ValueError(f"Received no valid parameters data from sys.stdin:\n{stdin_data}")
```